dreammall-earth/dreammall.earth
# fix(other): add missing account data

Motivation
----------
This is the data provided by our customers in our demo meeting.

![image](https://github.com/user-attachments/assets/1da33b52-b16a-4449-91c4-ca1a48422c98)


How to test
-----------
1. Check the deployment https://add-missing-account-data.git.dreammall.earth/
